Mathlib/Algebra/Ring/Opposite.lean

```diff
@@ -74,13 +74,6 @@ instance instCommRing [CommRing α] : CommRing αᵐᵒᵖ where
 instance instIsDomain [Ring α] [IsDomain α] : IsDomain αᵐᵒᵖ :=
   NoZeroDivisors.to_isDomain _
 
-instance instGroupWithZero [GroupWithZero α] : GroupWithZero αᵐᵒᵖ where
```

Comment:
Ooh, that's definitely a good upstreaming!

Reply:
Performed in #22323 already 😉